proposals/0272-sdl-javascript-manager-layer.md

```diff
@@ -686,16 +686,16 @@ class ScreenManagerBase extends SubManagerBase {
     presentChoiceSet (choiceSet, mode)
 
     /**
-     * Presents a keyboard on the Head unit to capture user input
-     * @param {String} initialText - The initial text that is used as a placeholder text. It might not work on some head units.
+     * Presents a keyboard on the head unit to capture user input
+     * @param {String} initialText - The initial text that is used as a placeholder text. It might not work on some head units
      * @param {KeyboardProperties} customKeyboardProperties - the custom keyboard configuration to be used when the keyboard is displayed
      * @param {KeyboardListener} keyboardListener - A keyboard listener to capture user input
-     * @return {Number} - A unique cancelID that can be used to cancel this keyboard. If `null`, no keyboard was created.
+     * @return {Number} - A unique cancelID that can be used to cancel this keyboard. If `null`, no keyboard was created
      */
     presentKeyboard (initialText, customKeyboardProperties, keyboardListener)
 
     /**
-     * Set a custom keyboard configuration for this session. If set to null, it will reset to default keyboard configuration.
+     * Set a custom keyboard configuration for this session. If set to null, it will reset to default keyboard configuration
```

Comment:
```suggestion
     * Set a custom keyboard configuration for this session. If set to null, it will reset to default keyboard configuration.
```
See above